Add `window.blur` config option (#7327)
alacritty/alacritty: CHANGELOG.md

- Support startup notify protocol to raise initial window on Wayland/X11
- Debug option `prefer_egl` to prioritize EGL over other display APIs
- Inline vi-mode search using `f`/`F`/`t`/`T`

> - `window.blur` config option to request blur behind the window

Reviewer: ```suggestion
- `window.blur` config option to request blur for transparent windows
```
